tests/playwright/src/specs/stress-test-ui.spec.ts

```diff
@@ -0,0 +1,74 @@
+/**********************************************************************
+ * Copyright (C) 2023-2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import type { ImagesPage } from '../model/pages/images-page';
+import { NavigationBar } from '../model/workbench/navigation';
+import { expect as playExpect, test } from '../utility/fixtures';
+import { waitForPodmanMachineStartup, waitWhile } from '../utility/wait';
+
+test.beforeAll(async ({ runner, welcomePage, page }) => {
+  runner.setVideoAndTraceName('stress-test-ui-e2e');
+  await welcomePage.handleWelcomePage(true);
+  await waitForPodmanMachineStartup(page);
+  // wait giving a time to podman desktop to load up
+  let images: ImagesPage;
+  try {
+    images = await new NavigationBar(page).openImages();
+  } catch (error) {
+    await runner.screenshot('error-on-open-images.png');
+    throw error;
+  }
+  await waitWhile(async () => await images.pageIsEmpty(), {
+    sendError: false,
+    message: 'Images page is empty, there are no images present',
+  });
+});
+
+test.afterAll(async () => {
+  test.setTimeout(90000);
+});
+
+test.describe
+  .serial('Verification of UI handling lots of objects', () => {
+    test(`Verification of images`, async ({ navigationBar }) => {
+      test.setTimeout(90_000);
+      const images = await navigationBar.openImages();
```

Comment:
Part of every test should be a "fast" check of the number of the rows in the table. counts for images, containers and pods the same.